samples/security/spire/curl-spire.yaml

```diff
@@ -13,66 +13,58 @@
 #   limitations under the License.
 
 ##################################################################################################
-# Notsleep service - based on the sleep service but has its own identity and affinity rule
+# curl service
 ##################################################################################################
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: notsleep
+  name: curl
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: notsleep
+  name: curl
   labels:
-    app: notsleep
-    service: notsleep
+    app: curl
+    service: curl
 spec:
   ports:
   - port: 80
     name: http
   selector:
-    app: notsleep
+    app: curl
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: notsleep
+  name: curl
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: notsleep
+      app: curl
   template:
     metadata:
       labels:
-        app: notsleep
+        app: curl
+        spiffe.io/spire-managed-identity: "true"
```

Comment:
What's this?

`notsleep` is used nowhere in the SPIRE docs AFAIR, and is only passingly used in  the ambient bookinfo guide as a different `sleep` deployment?

Do we even need it? What's it got to do with SPIRE?

Reply:
Additionally this would not meet the typical use Ben noted of different identity from sleep (now curl). Should it be `notcurl` instead?

Reply:
Oh, I see. I think I remember looking at this. I still don't think we need this tho?

It's just `curl` with a new label/annotation. The doc can use the regular one and label/annotate inline?

Reply:
Yep yep, always fine.

Reply:
https://github.com/istio/istio.io/issues/15754